{{cookiecutter.github_repository}}/settings/common.py

```diff
@@ -41,6 +41,7 @@
     "raven.contrib.django.raven_compat",
 {%- endif %}
     "mail_templated",  # https://github.com/artemrizhov/django-mail-templated
+    "django_extensions",  # http://django-extensions.readthedocs.org/
```

Comment:
Any reason for moving Django extensions to common settings from dev settings?

Reply:
@CuriousLearner generating db schema requires django extensions. Deployment environments uses production settings. So, to make django extensions available we have to initialize django with development settings inside the script.

After discussion with @theskumar we decided to keep django extensions in common. It'll reduce the sccript code and usually all of us needs django extensions on server as well while debugging. It'll help in that as well.

Reply:
Cool! Sounds good